### ISSUE TITLE
Maven cache error: AAR for react-native-reanimated has been found, FAILURE: Build failed with an exception

### ISSUE THREAD

**Issue by camnpr** [expo/expo-cli#4522]
### Summary

`eas build -p android --profile preview`

during to build apk use eas, report error happend step of `Run gradlew`.

before is normal, same configuration at yesterday.

### Environment

expo-env-info 1.0.5 environment info:
    System:
      OS: Windows 10 10.0.17134
    Binaries:
      Node: 14.17.0 - D:\Program Files\nodejs\node.EXE
      Yarn: 1.22.4 - ~\AppData\Roaming\npm\yarn.CMD
      npm: 6.14.13 - ~\AppData\Roaming\npm\npm.CMD
    IDEs:
      Android Studio: Version  3.0.0.0 AI-171.4408382
    npmPackages:
      @expo/webpack-config: 0.17.2 => 0.17.2
      expo: ~46.0.8 => 46.0.8
      react: 18.0.0 => 18.0.0
      react-dom: 18.0.0 => 18.0.0
      react-native: 0.69.4 => 0.69.4
      react-native-web: 0.18.7 => 0.18.7
    Expo Workflow: bare

### Please specify your device/emulator/simulator platform, model and version

build android

### Error output

✖ Build failed

🤖 Android build failed:
Maven cache error
Gradle build failed with unknown error. Please see logs for the "Run gradlew" phase.

> Configure project :react-native-reanimated
AAR for react-native-reanimated has been found
/home/expo/workingdir/build/AppExpo/node_modules/react-native-reanimated/android/react-native-reanimated-69-jsc.aar
[stderr] FAILURE: Build failed with an exception.
[stderr] * What went wrong:
[stderr] Could not determine the dependencies of task ':app:lintVitalReportRelease'.
[stderr] > Could not resolve all task dependencies for configuration ':app:releaseRuntimeClasspath'.
[stderr]    > Could not find com.google.android:cameraview:1.0.0.
[stderr]      Searched in the following locations:
[stderr]        - http://maven-cache-service.production.svc.cluster.local:8081/artifactory/libs-release/com/google/android/cameraview/1.0.0/cameraview-1.0.0.pom
[stderr]        - file:/home/expo/.m2/repository/com/google/android/cameraview/1.0.0/cameraview-1.0.0.pom
[stderr]        - file:/home/expo/workingdir/build/AppExpo/node_modules/react-native/android/com/google/android/cameraview/1.0.0/cameraview-1.0.0.pom
[stderr]        - file:/home/expo/workingdir/build/AppExpo/node_modules/jsc-android/dist/com/google/android/cameraview/1.0.0/cameraview-1.0.0.pom
[stderr]        - https://dl.google.com/dl/android/maven2/com/google/android/cameraview/1.0.0/cameraview-1.0.0.pom
[stderr]        - https://repo.maven.apache.org/maven2/com/google/android/cameraview/1.0.0/cameraview-1.0.0.pom
[stderr]        - https://www.jitpack.io/com/google/android/cameraview/1.0.0/cameraview-1.0.0.pom
[stderr]      Required by:
[stderr]          project :app > project :expo > project :expo-camera
[stderr] * Try:
[stderr] > Run with --stacktrace option to get the stack trace.
[stderr] > Run with --info or --debug option to get more log output.
[stderr] > Run with --scan to get full insights.
[stderr] * Get more help at https://help.gradle.org
[stderr] BUILD FAILED in 4m 3s
Deprecated Gradle features were used in this build, making it incompatible with Gradle 8.0.
You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.
See https://docs.gradle.org/7.3.3/userguide/command_line_interface.html#sec:command_line_warnings
6 actionable tasks: 6 executed

### Reproducible demo or steps to reproduce from a blank project

run `eas build -p android --profile preview`

```json
{
  "cli": {
    "version": ">= 0.60.0"
  },
  "build": {
    "development": {
      "distribution": "internal",
      "android": {
        "gradleCommand": ":app:assembleDebug"
      },
      "ios": {
        "buildConfiguration": "Debug"
      }
    },
    "preview": {
      "distribution": "internal",
      "android": {
        "buildType": "apk"
      }
    },
    "production": {}
  },
  "submit": {
    "production": {}
  }
}
```

[build error](https://expo.dev/accounts/camnpr/projects/RaodaorPoetry/builds/22b66e4a-d278-496e-a821-9fd2266953fd)

**Comment by otekdo** [comment]
I have the same issue. I set up the app with the Ignite boilerplate ``V8`` ``(Including Expo Preset)``
https://github.com/infinitered/ignite

These are the error logs i get from the EAS console
![image](https://user-images.githubusercontent.com/35440957/194462099-10c7c9d5-badc-431a-86d3-5206e4042e87.png)



Command
``eas build -p android --profile preview``

eas.json
```
{
  "cli": {
    "version": ">= 2.2.1"
  },
  "build": {
    "preview": {
      "android": {
        "buildType": "apk"
      }
    },
    "preview2": {
      "android": {
        "gradleCommand": ":app:assembleRelease"
      }
    },
    "preview3": {
      "developmentClient": true
    },
    "production": {}
  },
  "submit": {
    "production": {}
  }
}
```

**Comment by camnpr** [comment]
@otekdo you try delete file android and eas.json, run `npx expo prebuild` to generate need file, last exec `eas build -p android --profile preview`.
my env info is expo sdk 46.
good luck!

**Comment by otekdo** [comment]
@camnpr Indeed, it worked, i just needed to delete the ``.android`` folder, thanks
